README.md

```diff
@@ -127,16 +127,18 @@ global.instancename = dev-from-ide
 
 # DE4A Kafka settings
 de4a.kafka.enabled=false
+# Enables Kafka connection via HTTP (it must be used when HTTP connections are being established via proxy)
+de4a.kafka.http.enabled=false
 # Kafka server address (default: de4a-dev-kafka.egovlab.eu:9092)
 de4a.kafka.url=de4a-dev-kafka.egovlab.eu:9092
```

Comment:
Maybe add a commented out setting that looks like
`# de4a.kafka.url=https://de4a-dev-kafka.egovlab.eu`
And add another comment saying to uncomment this and remove the above one if HTTP mode is enabled. WIll make it simpler for pilots in this situation